### DISCUSSION TITLE
updating suricata rules in Airgap networks

### DISCUSSION THREAD

**Discussion by sinnetll19** [Security-Onion-Solutions/securityonion#2548] [Unsupported Versions] [answered]
For Airgap networks that do not have the ability to download suricata rule-sets from the internet what file path can we put the tarball file with the newly download rules (acquired from a different machine) in order for so-rule-update to pull from those rules and make them the all.rules file used by suricata.

**Comment by TOoSmOotH** [comment, marked as answer]
Untar and put it in /nsm/repo/rules/ named emerging-all.rules then run so-rule-update

**Reply by sinnetll19** [reply]
thank you. how can i view the version of Suricata that is installed?

**Reply by TOoSmOotH** [reply]
Should be 5.0.4. We will be moving to Suricata 6 with the next feature release in February.

**Reply by sinnetll19** [reply]
After placing the file under /nsm/repo/rules/ with the name emerging-all.rules and running so-rule-update nothing was changed. if removing the file emerging-all.rules and running so-rule-update also nothing was changed.
